pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -332,15 +335,66 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	)
 
 	Context("with VirtualMachineInstance metadata", func() {
-
+		table.DescribeTable(
+			"Should allow VMI creation with kubevirt.io/ labels only for kubevirt service accounts",
```

Comment:
What about "Should allow VMI creation with restricted kubevirt.io/ labels only ...."?

Reply:
@xpivarc Already included these tests, allow creation by service accounts